web/static/assets/hours_slept.txt

```diff
@@ -0,0 +1,48 @@
+0.5
+1.0
+1.5
+2.0
+2.5
+3.0
+3.5
+4.0
+4.5
+5.0
+5.5
+6.0
+6.5
+7.0
+7.5
+8.0
+8.5
+9.0
+9.5
+10.0
+10.5
+11.0
+11.5
+12.0
+12.5
+13.0
+13.5
+14.0
```

Comment:
at what point does this scale become redundant? 
or an alt question: how many people sleep more than 14h unless they are in rehab?